src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/EC/ECKeyFileTests.cs

```diff
@@ -177,6 +177,7 @@ public void ReadNistP521EncryptedPkcs8_Pbes2_Aes128_Sha384()
         public void ReadNistP521EncryptedPkcs8_Pbes2_Aes128_Sha384_PasswordBytes()
         {
             // PBES2, PBKDF2 (SHA384), AES128
+            // [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Unit test certificate")]
```

Comment:
I can't find better solution for this case. When `password` keyword is followed by base64string it is reported. In this case the method name contains `password` word